manifests/config_mountpoints.pp

```diff
@@ -0,0 +1,17 @@
+class lvm::config_mountpoints (
+  $mountpoints        = $::lvm::mountpoints,
+) {
+
+  if ($mountpoints) {
+
+    $mountpoint_defaults = {
+      ensure => directory,
+      mode   => '0755',
+      owner  => 'root',
+      group  => 'root',
+    }
```

Comment:
For default values probably something more like 750 would be better. You can always add a parameter if those need to be more configurable.